src/Entity/WSCallInterface.php

```diff
@@ -24,6 +24,11 @@ public function getEndpoint();
    */
   public function call($method = NULL, $replacements = [], $data = NULL, $options = [], $key = NULL, $tokens = []);
 
+  /**
+   * Retrieve information about the last call made.
+   */
+  public function lastCallStatus();
```

Comment:
Any unintended risks with storing the last call data? Like disclosing info about a user or something?

Reply:
Or lets say we made a call about a CC transaction or something?

Reply:
I was worried about the memory footprint of keeping this data, since we immediately make copies of it.  I'll adjust this to only include actual status otf debugging is off.